airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -36,32 +36,52 @@ class KubernetesHook(BaseHook):
     """
     Creates Kubernetes API connection.
 
+    - use in cluster configuration by using `extra__kubernetes__in_cluster` in connection
+    - use custom configuration either by providing content of kubeconfig file via
+     `extra__kubernetes__kube_config` in connection
+    - use custom config by provideing path to the file using `extra__kubernetes__kube_config_path`
+    - use default config by providing no extras
```

Comment:
```suggestion
    - use in cluster configuration by using `extra__kubernetes__in_cluster` in connection
    - use custom configuration either by providing content of kubeconfig file via
      `extra__kubernetes__kube_config` in connection
    - use custom config by provideing path to the file using `extra__kubernetes__kube_config_path`
    - use default config by providing no extras
```
Can you update the guide also and add cross-reference between the hook and guide?
https://airflow.readthedocs.io/en/latest/howto/connection/kubernetes.html